obcsim/obcsim_configuration.hpp

```diff
@@ -49,6 +49,12 @@
 #ifndef MSP_OP_SEND_CUBES_GATEWARE_CONF
 #define MSP_OP_SEND_CUBES_GATEWARE_CONF 0x77
 #endif
+#ifndef MSP_OP_SEND_NVM_CITI_CONF
+#define MSP_OP_SEND_NVM_CITI_CONF 0x78
+#endif
+#ifndef MSP_OP_SELECT_NVM_CITI_CONF
+#define MSP_OP_SELECT_NVM_CITI_CONF 0x79
+#endif
```

Comment:
These will conflict with what has already been added by @IHamed98 as part of his PR, which has already been merged. Please use `0x79` and `0x7A`.